<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MNTGenerationDialogForm</class>
 <widget class="QDialog" name="MNTGenerationDialogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate DTM</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_9">
   <item>
    <widget class="QLabel" name="m_MNTGenerationTitleLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="text">
      <string>Allows to generate a digital terrain model.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_inputGroupBox">
     <property name="title">
      <string>Input Layer</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QLabel" name="m_layerLabel">
        <property name="text">
         <string>Layer Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="m_layersComboBox"/>
      </item>
      <item>
       <widget class="QStackedWidget" name="m_inputstackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QCheckBox" name="m_dummycheckBox">
              <property name="text">
               <string>Dummy Value</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_dummylineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>9</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="m_Zlabel">
            <property name="text">
             <string>Quota Column:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_ZcomboBox"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_interpolatorGroupBox">
     <property name="title">
      <string>Interpolator</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QComboBox" name="m_interpolatorComboBox"/>
      </item>
      <item>
       <widget class="QStackedWidget" name="m_interparamStackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="m_averagePage">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="m_powerLabel">
            <property name="text">
             <string>Power:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="m_radiusLabel">
            <property name="text">
             <string>Radius:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="m_powerComboBox"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="m_radiusLineEdit"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="m_splinePage">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="m_sepXLabel">
              <property name="text">
               <string>Separation in X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_sepXSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QLabel" name="m_sepYLabel">
              <property name="text">
               <string>Separation in Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_sepYSpinBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="value">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="m_minPtsLabel">
              <property name="text">
               <string>Minimum Points:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="m_minPtsSpinBox">
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
              <property name="value">
               <number>20</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="m_overLabel">
              <property name="text">
               <string>Overlapping:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="m_overDoubleSpinBox">
              <property name="value">
               <double>5.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="m_mitasovaPage">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="m_tensionLabel">
              <property name="text">
               <string>Tension:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="m_tensionLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="m_smothLabel">
              <property name="text">
               <string>Smothness:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="m_smothLineEdit"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="m_minPtsMitLabel">
              <property name="text">
               <string>Minimum Points:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="m_minPtsMitLineEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_outputGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="m_rasterGroupBox">
        <property name="title">
         <string>Raster Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="m_resXLabel">
           <property name="toolTipDuration">
            <number>-1</number>
           </property>
           <property name="text">
            <string>Resolution: X</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="m_resXLineEdit"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="m_resYLabel">
           <property name="text">
            <string>Y</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="m_resYLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="m_dimCLabel">
           <property name="text">
            <string>Dimension: C</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="m_dimCLineEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="m_dimLLabel">
           <property name="text">
            <string>L</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="m_dimLLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="m_repositoryLabel">
        <property name="text">
         <string>Repository:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="m_repositoryLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_targetFileToolButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="m_targetDatasourceToolButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>BD</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_layerNameLabel">
        <property name="text">
         <string>Layer Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_newLayerNameLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="m_okPushButton">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="te::qt::widgets::HelpPushButton" name="m_helpPushButton">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="m_cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>te::qt::widgets::HelpPushButton</class>
   <extends>QPushButton</extends>
   <header>terralib/qt/widgets/help/HelpPushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
